.github/workflows/build.yml

```diff
@@ -0,0 +1,48 @@
+name: build
+
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install required packages
+        run: |
+          sudo apt install --no-install-recommends -y git cmake ninja-build gperf \
+            ccache dfu-util device-tree-compiler wget \
+            python3-dev python3-pip python3-setuptools python3-tk python3-wheel xz-utils file \
+            make gcc gcc-multilib g++-multilib libsdl2-dev libmagic1 python3-venv
+
+      - name: Install Zephyr
+        run: |
+          python3 -m venv $GITHUB_WORKSPACE/zephyrproject/.venv
+          source $GITHUB_WORKSPACE/zephyrproject/.venv/bin/activate
+          pip install west
+          west init -m https://github.com/eurus-project/zephyr $GITHUB_WORKSPACE/zephyrproject/ > /dev/null 2>&1
```

Comment:
What is the reason behind redirecting to `stderr`?

Reply:
I redirected both `stdout` and `stderr` of two commands with excessively large outputs, which made the stage logs difficult to read. These commands are part of the Zephyr installation process, and if they fail, the failure will still be reported.

To investigate errors of those commands, you can, for example, create a separate branch with commit that removes `> /dev/null 2>&1`. These investigation branches are not intended for merging into the main project and can be deleted after troubleshooting.

Reply:
I don't think we should do this, large outputs are not a problem as on GitHub you have sorted output step-wise. Also, needing to create separate branches to diagnose this adds a lot of friction to debugging when something fails.

Reply:
I honestly find 10.000+ repeated lines of `Updating repository(x/y)` really annoying and scrolling through them while working on this was painful.
I assume that, It is better to put those commands in separate steps - As you said. GitHub will sort them.

Reply:
That is a good solution IMO. Personally wheneve I've had to deal with CI logs, searching for `error` would have been enough, so the number of lines never bothered me.

Reply:
I am going to continue working on this - to provide better organization of steps

I am marking this PR as **draft**, it shouldn't be merged until this process is completed